content/kubermatic/master/architecture/concept/kkp-concepts/resource-quotas/_index.en.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "Resource Quotas"
+date = 2022-08-16T14:07:15+02:00
+weight = 20
+
++++
+
+## Resource Quotas in KKP
+
+Resource Quotas allow admins to set quotas on the amount of resources a subject can use. For now the only
+subject which is supported is Project, so the resource quotas currently limit the amount of resources that can be used project-wide.
+
+The resources in question are the resources of the user cluster:
+- CPU - the cumulated CPU used by the nodes on all clusters. 
+- Memory - the cumulated RAM used by the nodes on all clusters.
+- Storage - the cumulated disk size of the nodes on all clusters.
+
+This feature is available in the EE edition only.
+
+{{% notice note %}}
+**Note:** Do not confuse with the Resource Filter setting in the `Defaults And Limits` admin panel page. 
+That one just controls the size of the machines suggested to users in the KKP Dashboard during the cluster creation.
+{{% /notice %}}
+
+
+### Setting up the resource quotas
+
+The resource quotas are managed by admins either through the KKP UI or through the Resource Quota CRDs.
+
+Example ResourceQuota:
+```yaml
+apiVersion: kubermatic.k8c.io/v1
+kind: ResourceQuota
+metadata:
+  labels:
+    subject-kind: project
+    subject-name: tjqjkphnm6
```

Comment:
I wonder if we need to add labels in this example. Those are being added by the controller and removing them would simplify the example.